### ISSUE TITLE
No Content Shows when 'ViewPagerAndroid' wrapped by 'ViewPagerAndroid',fixed width$height given

### ISSUE THREAD

**Issue by jackyanjiaqi** [facebook/react-native#3807]
```
var ViewPagerTest = React.createClass({
    render:function(){
        return (
            <ViewPagerAndroid style={{flex:1}} initialPage={0}>
                <View>
                  <ViewPagerAndroid style={{width:300,height:400,backgroundColor:'#443213'}} initialPage={0} onPageScroll={e=>{console.log(e.nativeEvent.position);console.log(e.nativeEvent.offset)}>
                      <View style={styles.text}>
                          <Text>First child page</Text>
                      </View>
                      <View style={styles.text}>
                          <Text>Second child page</Text>
                      </View>
                  </ViewPagerAndroid>
                </View>
                <View style={styles.text}>
                    <Text>First page</Text>
                </View>
                <View style={styles.text}>
                    <Text>Second page</Text>
                </View>
            </ViewPagerAndroid>
          );
    }
});
...
var styles={
      text:{
      fontSize:20,
      backgroundColor:'#ffffff'
     }
}
```
__Cannot show "First child page" and "Second child page" even if the exact width and height are given.But the touchevent-related Callbacks invoked like onPageScroll and onPageSelected__

**Comment by browniefed** [comment]
Is this still happening?

**Comment by christopherdro** [comment]
@facebook-github-bot no-reply

**Comment by chrisciszak** [comment]
This problem still occurs, but I found a workaround, nasty but it's working.

I think there is some kind of defect which causes the inner ViewPagerAndroid not being shown on the initial render. However I noticed when I change the height of the element in my script editor, it re-renders the inner view correctly and is visible without problem. This is a nasty workaround but it's the only way I have managed to work around this issue. If someones has a better idea of achieve then same, would love to see it.

In the inner ViewPagerAndroid add the following:
`componentDidMount() { 
 setInterval(() => this.setState({height: height, width: width}), 0); 
}`
